miner/worker.go

```diff
@@ -1079,71 +1079,25 @@ func (w *worker) fillTransactions(interrupt *int32, env *environment) error {
 }
 
 // generateWork generates a sealing block based on the given parameters.
-func (w *worker) generateWork(genParams *generateParams) (*types.Block, error) {
-	work, err := w.prepareWork(genParams)
+func (w *worker) generateWork(params *generateParams) (*types.Block, error) {
```

Comment:
Reverting this back to `params` since upstream also uses that naming. Even though it collides with an import. We don't use that import in this function, so it should be fine.